app/models/user.rb

```diff
@@ -161,6 +161,6 @@ def legacy_otp_secret
     #    required for streaming cipher modes but including it is a best practice
     #    so that your code will continue to function correctly even if you later
     #    change to a block cipher mode.
-    cipher.update(cipher_text) + cipher.final
+    cipher.update!(cipher_text) + cipher.final
```

Comment:
This is a false positive - `cipher` isn't an Active Record instance

Reply:
Oh yeah, I've ignored that locally but apparently missed committing it.